.circleci/config.yml

```diff
@@ -130,8 +250,8 @@ jobs:
     executor: win/default
```

Comment:
### Recommendation from CircleCi
Use parallelism for faster test runs

### Conclusion
Skipping for now. It doesn't seem like there is a lot of benefit for time investment and circleci resources. Right now on average our unit-tests take about ~9-10 minutes. Setting up the VM to run the unit-tests on takes a little over 7 minutes. When enabling parallelism this time for setup actually increases (since we are using additional VMs for each executor that run simultaneously). As we continue to add more tests, this might be something that we want to revisit in the future.